recipe/meta.yaml

```diff
@@ -51,3 +52,4 @@ extra:
     - jasongrout
     - maartenbreddels
     - pbugnion
+    - jtpio
```

Comment:
Are you ok being added as [a maintainer]( https://conda-forge.org/docs/orga/joining-the-team.html#feedstock-maintainer-responsibilities ) here @jtpio? 🙂

Reply:
I think it's probably late where @jtpio is. I'll take out this addition for now, and we can add it back in a build bump, so we don't need to delay for his ack.